src/classes/Recipe.js

```diff
@@ -0,0 +1,35 @@
+class Recipe {
+    constructor(recipeDetails) {
+        this.id = recipeDetails.id;
+        this.name = recipeDetails.name;
+        this.image = recipeDetails.image;
+        this.portions = recipeDetails.portions;
+        this.instructions = recipeDetails.instructions;
+        this.tags = recipeDetails.tags;
+
+    }
+    showIngredientsNeeded() {
+        return this.portions.map((portion) => {
+            return portion.name;  
+        });
+    }
+    calcTotalIngredientCost() {
```

Comment:
I'm thinking `getPortionCosts` for this one

Reply:
Got it changed over